circuits/web/http.py

```diff
@@ -22,22 +23,9 @@
 from .url import parse_url
 from .utils import is_unix_socket
 
-try:
-    from urllib.parse import quote
-    from urllib.parse import urlparse, urlunparse
-except ImportError:
-    from urllib import quote  # NOQA
-    from urlparse import urlparse, urlunparse  # NOQA
-
-
 MAX_HEADER_FRAGENTS = 20
 HTTP_ENCODING = 'utf-8'
 
-try:
-    unicode
-except NameError:
-    unicode = str
```

Comment:
`unicode` was not used anywhere in this file and I do not think it was added as "utility" for other modules.